rental.rb

```diff
@@ -0,0 +1,18 @@
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Great job using getters for your book and person. however, please kindly take note that the project requirement is to use Getters and Setters for `@book and @person instance variables`. please kindly make use of attr_accessor for your `@book and @person instance variables`.